docs/framework/deployment/reducing-system-restarts.md

```diff
@@ -10,7 +10,7 @@ author: "mairaw"
 ms.author: "mairaw"
 ---
 # Reducing System Restarts During .NET Framework 4.5 Installations
-The .NET Framework 4.5 installer uses the [Restart Manager](https://go.microsoft.com/fwlink/?LinkId=231425) to prevent system restarts whenever possible during installation. If your app setup program installs the .NET Framework, it can interface with the Restart Manager to take advantage of this feature. For more information, see [How to: Get Progress from the .NET Framework 4.5 Installer](how-to-get-progress-from-the-dotnet-installer.md).  
+The .NET Framework 4.5 installer uses the [Restart Manager](https://docs.microsoft.com/windows/win32/rstmgr/about-restart-manager) to prevent system restarts whenever possible during installation. If your app setup program installs the .NET Framework, it can interface with the Restart Manager to take advantage of this feature. For more information, see [How to: Get Progress from the .NET Framework 4.5 Installer](how-to-get-progress-from-the-dotnet-installer.md).  
```

Comment:
```suggestion
The .NET Framework 4.5 installer uses the [Restart Manager](/windows/win32/rstmgr/about-restart-manager) to prevent system restarts whenever possible during installation. If your app setup program installs the .NET Framework, it can interface with the Restart Manager to take advantage of this feature. For more information, see [How to: Get Progress from the .NET Framework 4.5 Installer](how-to-get-progress-from-the-dotnet-installer.md).  
```